packer/README.md

```diff
@@ -1,10 +1,147 @@
-# Packer Build
+# Starter-Kit-SEA-Modernization
+
+## Steps to build Starter-Kit-SEA-Modernization Automated Machine Image
+
+This README outlines the steps for creating a virtual hard disk image containing
+an evaluation copy of Windows 2019 Server, with SQL Server 2019 Express edition,
+SQL Server Management Studio, Google Chrome, Dot Net Framework 4.8, the Dot Net
+Core SDK, and NuGet Package Manager.
+
+## Quick Start
+
+If you want to jump ahead, there are three steps
+
+- Install the prerequisites: Hyper-V and Packer
+- Clone the Starter-Kit-SEA-Modernization repository from Ed-Fi-Alliance-OSS on
+  Github
+- Run the build.ps1 as an Admin user
+
+## Step by step
+
+### Clone the repo
+
+Clone the [Starter-Kit-SEA-Modernization repository](https://github.com/Ed-Fi-Alliance-OSS/Starter-Kit-SEA-Modernization/)
+
+### Turn on Windows features for Hyper-V
+
+  You will need both sub-items: Hyper-V Platform and Hyper-V Management Tools.
+  For directions on enabling these features, follow the directions found
+  [here](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v#enable-the-hyper-v-role-through-settings)
 
-Execute `./build-vm.ps1 -PackerFile .\win2019-eval-base.pkr.hcl -VariablesFile .\base-variables.json` to generate the base image. Folders that are created during the build process are `./build` and `./dist`.
-Execute `./build-vm.ps1 -PackerFile .\sea-starter-kit-win2019-eval.pkr.hcl -VariablesFile .\starter-kit-variables.json` to generate the start kit image. Folders that are created during the build process are `./build` and `./dist`.
+### Download an install Packer
+
+```powershell
+choco install Packer
+```
+
+### Open a PowerShell console (greater or equal to PSVersion 5) in elevated mode
+
+Set your location in the console to the Ed-Fi-Starter-Kit-Assessments root folder.
```

Comment:
change to Starter-Kit-SEA-Data-Collection\packer folder

Reply:
done